go.mod

```diff
@@ -2,7 +2,8 @@ module github.com/pion/dtls/v2
 
 require (
 	github.com/pion/logging v0.2.2
-	github.com/pion/transport v0.14.1
+	github.com/pion/transport v0.12.2
+	github.com/pion/transport/v2 v2.0.0
```

Comment:
This is definitely not right. We should not be downgrading the currently in use module version while also adding v2 despite the fact that it's not in use.

Reply:
Unfortunately, the renovate bot is too dumb to recognize that this is a major version change.
In my opinion, the bot not should attempt those upgrade anyway as a major version change usually entails breaking APIs.

Reply:
Yeah. I tried to get the folks behind the bot to understand that but no. My comment was originally about a different change, it included a downgrade of pion/transport to v0.12. That at least seems to no longer be happening.